src/internal/components/configurationPanels/ScatterPlotConfigurationPanel.tsx

```diff
@@ -49,6 +50,13 @@ export default class ScatterPlotConfigurationPanel extends ConfigurationPanelCon
                         properties={properties}
                         pushData={pushData}
                     >
+                        <NameSubsection
+                            disabled={controlsDisabled}
+                            configPanelDisabled={controlsDisabled}
+                            axis={"xaxis"}
+                            properties={properties}
+                            pushData={pushData}
+                        />
```

Comment:
share some properties with NameSubsection

Reply:
I think we should not group it